README.md

```diff
@@ -239,20 +240,55 @@ Reference Code Structure
 
 ```bash
 
-├── CONTRIBUTING.md                               <-- How to contribute to Trapheus
+├── CHANGELOG.md
 ├── LICENSE.md                                    <-- The MIT license.
 ├── README.md                                     <-- The Readme file.
-├── events
-│   └── event.json                                <-- JSON event file to be used for local SAM testing
-├── screenshots                                   <-- Folder for screenshots of teh state machine.
-│   ├── Trapheus-logo.png
+├── docs                                          <-- The Readme files.
+│   ├── README.fr.md
+│   └── README.zh-CN.md
+├── install.py
+├── labs
```

Comment:
lets remove the  substructure under`labs` - because it is still a work in progress

Reply:
ok